lib/getLatestRelease.js

```diff
@@ -13,58 +14,79 @@ if (process.env.GH_TOKEN) {
 
 const getOS = str => {
   if (includes(str, 'darwin') || includes(str, '.dmg')) {
-    return 'OS X';
+    return 'macOS';
   } else if (includes(str, 'win32') || includes(str, '.exe')) {
     return 'Windows';
   } else if (includes(str, 'linux')) {
     return 'Linux';
   }
 };
 
-const getInstallerType = file => {
-  if (!includes(file, '.exe')) {
-    return '';
-  }
-
-  if (includes(file, 'Portable')) {
-    return '(Portable)';
-  } else {
-    return '(Installer)';
+const getArch = str => {
+  if (includes(str, '64') || getOS(str) === 'macOS') {
```

Comment:
IMHO this is a bit too generic - and it'll go horribly wrong if we get up to Etcher version 1.1.64 ? ;-)
What about:
```js
if (includes(str, 'x64') || includes(str, 'x86_64') || includes(str, 'amd64') || getOS(str) === 'macOS') {
```
(included `amd64` as forward-planning, see https://github.com/resin-io/etcher/releases )